src/test-utils/testing-library.tsx

```diff
@@ -5,19 +5,20 @@ import { render, type RenderOptions } from '@testing-library/react';
 import { TooltipProvider } from '@box/blueprint-web';
 import { IntlProvider } from 'react-intl';
 import { AutofillContextProvider } from '@box/metadata-editor';
+
 import { FeatureProvider } from '../elements/common/feature-checking';
 
 jest.unmock('react-intl');
 
 const Wrapper = ({
     children,
     features = {},
-    isAiSuggestionsFeatureEnabled = false,
     fetchSuggestions = () => Promise.resolve([]),
+    isAiSuggestionsFeatureEnabled = false,
 }) => (
     <AutofillContextProvider
-        isAiSuggestionsFeatureEnabled={isAiSuggestionsFeatureEnabled}
         fetchSuggestions={fetchSuggestions}
+        isAiSuggestionsFeatureEnabled={isAiSuggestionsFeatureEnabled}
     >
```

Comment:
after looking at this again and noticed we are passing props specifically for it,  think we need to consider moving this out of the testing-library into the metadata-editor tests directly, all the tests in the repo will use this wrapper eventually, i cant imagine that provider being used that much to warrant being in the wrapper.  the other providers where are more generic

Reply:
Agree. `AutofillContextProvider` is needed only for `MetadataInstanceEditor` tests.
I did tried moving it out but it resulted in failing tests
```
    TypeError: (0 , _reactIntl.useIntl) is not a function
```

Not quite sure what's happening. But I needed to move the whole wrapper component to `MetadataInstanceEditor` test.